### PR TITLE
add GOV ID to exid-types

### PR DESCRIPTION
**Pull request by tychonievich**
Register gov.genealogy.net, a historical gazeteer maintained by CompGen, using information provided by @albertemmerich

See also https://github.com/FamilySearch/GEDCOM-registries/pull/96

### CODE REVIEW

File: exid-types.json [hunk 1]
```diff
@@ -105,4 +105,12 @@
     "change-controller": "wikitree.com",
     "reference": "https://www.wikitree.com/wiki/Help:WikiTree_ID"
   },
+  {
+    "label": "GOV ID",
+    "type": "http://gov.genealogy.net/",
+    "description": "Historical Gazeteer place ID",
+    "contact": "gov-support@genealogy.net",
+    "change-controller": "CompGen",
+    "reference": "https://gedcom.io/exid-type/GOV-ID"
```
Comment (dthaler) [inline exid-types.json:114]: I get 404 not found at this URL

**Comment by tychonievich** [comment]
While uploading this, it occurred to me to wonder why we have both exid-types.json and [the uri/exid-type registry](https://github.com/FamilySearch/GEDCOM-registries/tree/main/uri/exid-types); the two appear to be fully redundant with one another.

File: exid-types.json (hunk 1 above)
Comment (dthaler) [reply exid-types.json:114]: How about using https://wiki.genealogy.net/GOV
(which is what "About the project" links to from http://gov.genealogy.net/search/name)

Comment (albertemmerich) [reply exid-types.json:114]: the proposed YAML file links to https://wiki.genealogy.net/GOV. However the YAML file ist not yet uploaded, so the 404 occurs.

Comment (tychonievich) [inline exid-types.json:114]:
```suggestion
    "reference": "https://wiki.genealogy.net/GOV"
```

**Comment by tychonievich** [comment]
Approved in steering committee 12 OCT 2024

**Pull request merged**